src/common/transformations/src/transformations/sdpa_to_paged_attention/state_management_pattern.cpp

```diff
@@ -45,21 +72,31 @@ ov::pass::StateManagementPattern::StateManagementPattern(ParameterVector& kv_par
                                                          Output<Node> max_context_len) {
     MATCHER_SCOPE(StateManagementPattern);
 
-    auto k_past_var = pattern::wrap_type<v6::ReadValue>({pattern::any_input()});
-    auto k_past_par = pattern::wrap_type<v0::Parameter>();
-    auto k_past = std::make_shared<pattern::op::Or>(
-        OutputVector{pattern::wrap_type<v8::Gather>({k_past_var, pattern::any_input(), pattern::any_input()}),
-                     k_past_par});
-    k_past = std::make_shared<pattern::op::Or>(
-        OutputVector{k_past,
-                     pattern::wrap_type<v1::Transpose>(
-                         {k_past, pattern::any_input()})});  // Transpose is used when kv-cache is stored in a not usual
-                                                             // layout, example: bloom
     auto k_current = pattern::any_input();
-    auto k_current2 = pattern::any_input();
-    auto k_current_reshaped = pattern::wrap_type<v1::Reshape>({k_current2, pattern::any_input()});
-    auto k_concat = pattern::wrap_type<v0::Concat>(
-        {k_past, std::make_shared<pattern::op::Or>(OutputVector{k_current_reshaped, k_current})});
+    std::shared_ptr<ov::Node> k_past_par, k_current2, k_concat, k_current_reshaped;
```

Comment:
maybe it's better to have a struct?

struct StructName {
std::shared_ptr<ov::Node> past_par, current2, concat, current_reshaped;
} K, V, KV;

K = kv_read_and_concat(k_current); // returns structure from this function
V = kv_read_and_concat(v_current);
KV = kv_read_and_concat(kv_current);